src/WebJobs.Script/Workers/WorkerStatus.cs

```diff
@@ -15,10 +15,5 @@ public class WorkerStatus
         /// Gets or sets the current latency for worker channel.
         /// </summary>
         public TimeSpan Latency { get; set; }
-
-        /// <summary>
-        /// Gets or sets the process statistics for the worker process.
-        /// </summary>
-        public ProcessStats ProcessStats { get; set; }
     }
 }
```

Comment:
Are the worker process stats are measured by WebJobs now?

Reply:
Yes, see the calls to IHostProcessMonitor.Register/UnregisterChildProcess in the WorkerProcess class. The IHostProcessMonitor service now monitors all processes - main as well as OOP worker processes